resolwe/flow/engines/__init__.py

```diff
@@ -79,25 +83,8 @@ def communicate(self, run_sync=False, verbosity=1):
                         data.save()
                         continue
 
-                    script_template = data.process.run.get('script', '')
-                    inputs = data.input.copy()
-                    hydrate_input_references(inputs, data.process.input_schema)
-                    # hydrate_input_uploads(inputs, data.process.input_schema)
-
-                    info = {}
-                    # info['case_ids'] = data.case_ids
-                    info['data_id'] = data.id
-                    # info['data_path'] = settings.RUNTIME['data_path']
-                    # info['slugs_path'] = settings.RUNTIME['slugs_path']
-                    inputs['proc'] = info  # add script info
-
-                    try:
-                        script = template.Template('{% load resource_filters %}{% load mathfilters %}' +
-                                                   script_template).render(template.Context(inputs))
-                    except template.TemplateSyntaxError as ex:
-                        data.status = Data.STATUS_ERROR
-                        data.process_error.append('Error in process script: {}'.format(ex))
-                        data.save()
+                    script = self.exprengines['dtlbash'].eval(data)
```

Comment:
The expression that the processor use will be defined on the processor. Currently we only support the ```dtlbash```, but the platform was designed to be general, so that we can add more expression engines later. I do not think we need FLOW_EXPRESSION_ENGINES in settings at all? We should just load all available expression engines. Because ```dtlbash``` is available and because it is hardcoded, an exception can not happen here. When we will support another expression engine and read its name from the processor, we can also add the error handling.